cli/src/lib/compiler/compile.js

```diff
@@ -94,8 +94,19 @@ const compile = async ({
     }
     const compileFile = async (source, destination) => {
         if (source.match(extensionPattern)) {
-            const result = await babel.transformFileAsync(source, babelConfig)
-            await fs.writeFile(destination, result.code)
+            try {
```

Comment:
Catching here allows us to recover from a syntax error and re-compile when a change is detected, which is nice for the developer.

Reply:
Confirmed this works in Node 16.13.0 👍

Reply:
Neato !